testsuite/features/step_definitions/command_steps.rb

```diff
@@ -156,6 +156,18 @@
   end
 end
 
+Then(/^file "([^"]*)" should contain "([^"]*)" on server$/) do |file, content|
```

Comment:
To avoid to maintain two methods, on negations I think we could use Cucumber optional params like that:
```suggestion
Then(/^file "([^"]*)" should( not)? contain "([^"]*)" on server$/) do |negate, file, content|
if negate:
...
else:
....
```
Just a suggestion :) Is fine anyway.

Reply:
Added optional param.